### PR TITLE
Remove typo

### PR DESCRIPTION
**Pull request by omimakhare**
## Short description
This Pull Request fixes a typographical error in the README.md file.

## List of changes proposed in this pull request
- Correct "pyshical" to "physical" in the README.md file.

## How to test
No need to test as they are just small typographical fixes.

## Additional Information:
This fix is not related to any existing issue. It's a minor typo that I noticed while reviewing the README.

Thank you for considering this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -423,10 +423,10 @@ $ yarn run-android
 $ yarn run-ios
 ```
 
-### Pyshical devices
-The React Native documentation provides a [useful guide](https://reactnative.dev/docs/running-on-device) for running projects on pyshical devices.
+### physical devices
```
Comment (LazyAfternoons) [inline README.md:426]: Missing capital letter.

File: README.md [hunk 2]
```diff
@@ -576,4 +576,4 @@ In the meantime, there are two complementary and simultaneous activities going o
 
 You can keep track of the latest developments by filtering all the PRs according to the type of activity:
 
-[![NativeBase's dismissal label](https://img.shields.io/github/labels/pagopa/io-app/NativeBase%20dismissal%20%F0%9F%A5%B7%F0%9F%8F%BC)](https://github.com/pagopa/io-app/labels/NativeBase%20dismissal%20%F0%9F%A5%B7%F0%9F%8F%BC) [![Design System's label](https://img.shields.io/github/labels/pagopa/io-app/Design%20System)](https://github.com/pagopa/io-app/labels/Design%20System)
\ No newline at end of file
+[![NativeBase's dismissal label](https://img.shields.io/github/labels/pagopa/io-app/NativeBase%20dismissal%20%F0%9F%A5%B7%F0%9F%8F%BC)](https://github.com/pagopa/io-app/labels/NativeBase%20dismissal%20%F0%9F%A5%B7%F0%9F%8F%BC) [![Design System's label](https://img.shields.io/github/labels/pagopa/io-app/Design%20System)](https://github.com/pagopa/io-app/labels/Design%20System)
```
Comment (LazyAfternoons) [inline README.md:579]: Why did this line change?

**Review by LazyAfternoons — CHANGES_REQUESTED** [review]
Comments

File: README.md (hunk 2 above)
Comment (dmnplb) [reply README.md:579]: Probably different end of line management

File: README.md (hunk 1 above)
Comment (omimakhare) [reply README.md]: Now added capital letter.

File: README.md (hunk 2 above)
Comment (omimakhare) [reply README.md:579]: Checked with difference checker but nothing is changed in this line.

Comment (Vangaorth) [reply README.md:579]: Hi @omimakhare, please discard such hunk in order for us to approve your PR

**Comment by dmnplb** [comment]
@LazyAfternoons @Vangaorth Let's approve this PR. We're sure the end of the README is the same in both versions, so we shouldn't be so strict

**Review by CrisTofani — APPROVED** [review]

**Pull request merged**